### PR TITLE
Add partial matching for find command

### LINKED ISSUE #310: [PE-D][Tester B] Find order does not match partial names
### Description:
Find order feature is less useful than it can be as as the user has to remember the full name of the client.

### Steps to Reproduce:
`find order Dav`

### Expected Behavior:
`David Li`'s order shown

### Observed Behavior:
`0 orders listed!`

### Screenshot(s):

-------------
Labels: `severity.Low` `type.FeatureFlaw`
original: Zhannyhong/ped#14

### PR DESCRIPTION
**Pull request by mihirniyogi**
Fixes #310

### CODE REVIEW

File: src/main/java/powerbake/address/commons/util/StringUtil.java [hunk 1]
```diff
@@ -35,7 +36,7 @@ public static boolean containsWordIgnoreCase(String sentence, String word) {
         String[] wordsInPreppedSentence = preppedSentence.split("\\s+");
 
         return Arrays.stream(wordsInPreppedSentence)
-                .anyMatch(preppedWord::equalsIgnoreCase);
+                .anyMatch(w -> w.toLowerCase().contains(preppedWord.toLowerCase()));
```
Comment (Emmanuel2001) [inline src/main/java/powerbake/address/commons/util/StringUtil.java:39]: Good Job! This makes our find feature more useful for the users 😃

**Review by Emmanuel2001 — COMMENTED** [review]
LGTM!

File: src/test/java/powerbake/address/commons/util/StringUtilTest.java [hunk 2]
```diff
@@ -127,11 +127,11 @@ public void containsWordIgnoreCase_validInputs_correctResult() {
         assertFalse(StringUtil.containsWordIgnoreCase("    ", "123"));
 
         // Matches a partial word only
-        assertFalse(StringUtil.containsWordIgnoreCase("aaa bbb ccc", "bb")); // Sentence word bigger than query word
+        assertTrue(StringUtil.containsWordIgnoreCase("aaa bbb ccc", "bb")); // Sentence word bigger than query word
         assertFalse(StringUtil.containsWordIgnoreCase("aaa bbb ccc", "bbbb")); // Query word bigger than sentence word
 
         // Matches word in the sentence, different upper/lower case letters
-        assertTrue(StringUtil.containsWordIgnoreCase("aaa bBb ccc", "Bbb")); // First word (boundary case)
+        assertTrue(StringUtil.containsWordIgnoreCase("aaa bBb ccc", "Aaa")); // First word (boundary case)
```
Comment (mogmyij) [inline src/test/java/powerbake/address/commons/util/StringUtilTest.java:134]: Nice making "find" match partial words and different cases will make powerbake more usable!

**Review by mogmyij — COMMENTED** [review]
LGTM!
